force_wfmanager/ui/review/plot.py

```diff
@@ -365,6 +304,16 @@ def _update_plot_data(self):
         self._plot_data.set_data('color_by', self._data_arrays[c_index])
 
         self.resize_plot()
```

Comment:
The `resize_plot` method should be moved from `_update_plot_data`.

OR

`_plot_updater_default` should call an alternative method that does not affect the plot view.

Reply:
I made it conditional. Through the timer's intermediation, it basically gets called iff there was a `analysis_model.evaluation_steps[]` event in the last second.